File: app/src/main/java/com/vn/ezlearn/utils/QuestionUtils.java
package com.vn.ezlearn.utils;

import com.vn.ezlearn.models.MyContent;

import java.util.List;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 13/10/2017.
 */

public class QuestionUtils {
    private static QuestionUtils questionUtils;
    private List<MyContent> myContentList;

    public static QuestionUtils getInstance() {
        if (questionUtils == null) {
            questionUtils = new QuestionUtils();
        }
        return questionUtils;
    }

    public void clearMyContentList() {
        myContentList = null;
    }

    public List<MyContent> getMyContentList() {
        return myContentList;
    }

    public void setMyContentList(List<MyContent> myContentList) {
        this.myContentList = myContentList;
    }
}

File: app/src/main/java/com/vn/ezlearn/utils/QuestionUtils.kt
package com.vn.ezlearn.utils

import com.vn.ezlearn.models.MyContent

/**
 * Created by FRAMGIA\nguyen.duc.manh on 13/10/2017.
 */

class QuestionUtils {
    var myContentList: List<MyContent>? = null

    fun clearMyContentList() {
        myContentList = null
    }

    companion object {
        private var questionUtils: QuestionUtils? = null

        val instance: QuestionUtils
            get() {
                if (questionUtils == null) {
                    questionUtils = QuestionUtils()
                }
                return questionUtils as QuestionUtils
            }
    }
}
